gson/src/test/java/com/google/gson/functional/FormattingStyleTest.java

```diff
@@ -35,7 +34,7 @@
 public class FormattingStyleTest {
 
   private static final String[] INPUT = {"v1", "v2"};
-  private static final String EXPECTED = "[<EOL><INDENT>\"v1\",<EOL><INDENT>\"v2\"<EOL>]";
+  private static final String EXPECTED = "[<EOL><INDENT>\"v1\",<SPACE><EOL><INDENT>\"v2\"<EOL>]";
```

Comment:
I misread this as saying that there would be a space before a newline here, which of course would be surprising. Perhaps replace `<SPACE><EOL>` with `<SPACE_OR_EOL>` or something like that?

Reply:
Have named them `COLON_SPACE` and `COMMA_SPACE`, still slightly misleading but hopefully a bit less than before. Are these fine?